schemachange/sc_struct.c

```diff
@@ -243,6 +243,8 @@ int pack_schema_change_protobuf(struct schema_change_type *s, void **packed_sc,
     s->spname_len = strlen(s->spname) + 1;
```

Comment:
I didn't update the packing/unpacking logic for the pre-protobuf schema change serialization protocol. Not sure if we want to keep that protocol in a working state so that we can rollback to it or if it's deprecated.